### ISSUE TITLE
Host static version on GitHub pages

### ISSUE THREAD

**Issue by docwhat** [docwhat/dianewillis#1]
Your mission: https://dianewillis.org should be hosted as a github page from this repo.

1. [x] Read up on GitHub pages.
1. [x] A static site is put together in the `static` branch in the `src/` directory.
1. [x] Make whatever changes you want to the repository except do not delete the git tags.
1. [x] Create a `README.md` with any applicable badges.  See the [current README](https://github.com/holtje/dianewillis/blob/34c529c906d8e9a5f8613db69b777b0fbbe7ec05/README.md) for an example.

I have lots of half-completed thoughts in here.  Feel free to remove anything you want.

The last 2 steps will be done together:

1. [x] Change the DNS entry for dianewillis.org to point to github pages.
1. [x] Host a redirector for dianewillis.com on gerf.org that redirects to dianewillis.org.

**Comment by RHoltje** [comment]
Hi,
I have some comments / questions:

 - I tried doing the GitHub tutorial for GitPages without much luck.  It started, but wouldn't move beyond step one in the tutorial.  ( The bot was supposed to respond after I did certain actions, but it didn't. )
- On the repo I cannot find the index.html file or the text documents.
- i don't understand how to move forward on this, need more guidance.
  since this is an old rails app, won't we just have to trash everything?
- you list "A static site is put together in the static branch in the src/ directory." The GitHub Pages documentation says that says that the site needs to be in either the main branch or the gthpages branch.
- GitHub pages uses Jekyll, which seems to be a Ruby gem.  Is that what you want?

**Comment by docwhat** [comment]
- > I tried doing the GitHub tutorial for GitPages without much luck. It started, but wouldn't move beyond step one in the tutorial. ( The bot was supposed to respond after I did certain actions, but it didn't. )

    I'll have to take a look but the tutorial works best on brand-new repositories.

- > On the repo I cannot find the index.html file or the text documents.

    There isn't any HTML.  It's generated from `README.md`

- >  don't understand how to move forward on this, need more guidance.  since this is an old rails app, won't we just have to trash everything?

    Yup.  That's already done in the `static` branch.

- > you list "A static site is put together in the static branch in the src/ directory." The GitHub Pages documentation says that says that the site needs to be in either the main branch or the gthpages branch.

    Yup.  The `static` branch needs to be merged into the `master` branch.

- > GitHub pages uses Jekyll, which seems to be a Ruby gem. Is that what you want?

    Sort of?  We can talk about this in person.
